[2022-04-07 19:24:51,944] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task scheduled__2017-04-01T00:00:00+00:00 [queued]>
[2022-04-07 19:24:52,005] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task scheduled__2017-04-01T00:00:00+00:00 [queued]>
[2022-04-07 19:24:52,005] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:24:52,005] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 19:24:52,006] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:24:52,281] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task> on 2017-04-01 00:00:00+00:00
[2022-04-07 19:24:52,308] {standard_task_runner.py:52} INFO - Started process 3943 to run task
[2022-04-07 19:24:52,332] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task', 'scheduled__2017-04-01T00:00:00+00:00', '--job-id', '12907', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmp0e4wj9nl', '--error-file', '/tmp/tmp4xn4u7xl']
[2022-04-07 19:24:52,334] {standard_task_runner.py:80} INFO - Job 12907: Subtask create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task
[2022-04-07 19:24:52,644] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task scheduled__2017-04-01T00:00:00+00:00 [running]> on host 2c54f2187010
[2022-04-07 19:24:52,896] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2017-04-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2017-04-01T00:00:00+00:00
[2022-04-07 19:24:52,898] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 19:24:52,952] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 19:24:52,998] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_posts_questions_bq_create_posts_questions_stack_overflow_data_partitioned_table_task_2017_04_01T00_00_00_00_00_8e30fd49019df585d5c783ce76fe7094
[2022-04-07 19:24:53,697] {taskinstance.py:1774} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2153, in execute
    job = self._submit_job(hook, job_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2119, in _submit_job
    retry=self.result_retry,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 439, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1560, in insert_job
    job.result(timeout=timeout, retry=retry)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1498, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1488, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 728, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/future/polling.py", line 135, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Could not cast literal "creation_date" to type TIMESTAMP at [1:111]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_posts_questions_bq_create_posts_questions_stack_overflow_data_partitioned_table_task_2017_04_01T00_00_00_00_00_8e30fd49019df585d5c783ce76fe7094

[2022-04-07 19:24:53,723] {taskinstance.py:1288} INFO - Marking task as FAILED. dag_id=load_to_BigQuery, task_id=create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task, execution_date=20170401T000000, start_date=20220407T192451, end_date=20220407T192453
[2022-04-07 19:24:53,753] {standard_task_runner.py:98} ERROR - Failed to execute job 12907 for task create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task (400 Could not cast literal "creation_date" to type TIMESTAMP at [1:111]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_posts_questions_bq_create_posts_questions_stack_overflow_data_partitioned_table_task_2017_04_01T00_00_00_00_00_8e30fd49019df585d5c783ce76fe7094
; 3943)
[2022-04-07 19:24:53,775] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-04-07 19:24:53,823] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-07 22:18:07,294] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task scheduled__2017-04-01T00:00:00+00:00 [queued]>
[2022-04-07 22:18:07,357] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task scheduled__2017-04-01T00:00:00+00:00 [queued]>
[2022-04-07 22:18:07,357] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:18:07,357] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 22:18:07,357] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:18:07,768] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task> on 2017-04-01 00:00:00+00:00
[2022-04-07 22:18:07,824] {standard_task_runner.py:52} INFO - Started process 6249 to run task
[2022-04-07 22:18:07,831] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task', 'scheduled__2017-04-01T00:00:00+00:00', '--job-id', '14165', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmpbpxw2i2t', '--error-file', '/tmp/tmpdbzfl5pm']
[2022-04-07 22:18:07,833] {standard_task_runner.py:80} INFO - Job 14165: Subtask create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task
[2022-04-07 22:18:08,115] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task scheduled__2017-04-01T00:00:00+00:00 [running]> on host 36107d30f64e
[2022-04-07 22:18:08,284] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2017-04-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2017-04-01T00:00:00+00:00
[2022-04-07 22:18:08,286] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 22:18:08,312] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 22:18:08,356] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_posts_questions_bq_create_posts_questions_stack_overflow_data_partitioned_table_task_2017_04_01T00_00_00_00_00_bc44c16dcc93f65a854d20f24cbe4b60
[2022-04-07 22:18:12,299] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=load_to_BigQuery, task_id=create_external_posts_questions.bq_create_posts_questions_stack_overflow_data_partitioned_table_task, execution_date=20170401T000000, start_date=20220407T221807, end_date=20220407T221812
[2022-04-07 22:18:12,384] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-07 22:18:12,551] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
